azuread/table_azuread_group.go

```diff
@@ -2,222 +2,430 @@ package azuread
-
-			// Json fields
-			{Name: "assigned_labels", Type: proto.ColumnType_JSON, Description: "The list of sensitivity label pairs (label ID, label name) associated with a Microsoft 365 group."},
-			{Name: "group_types", Type: proto.ColumnType_JSON, Description: "Specifies the group type and its membership. If the collection contains Unified, the group is a Microsoft 365 group; otherwise, it's either a security group or distribution group. For details, see [groups overview](https://docs.microsoft.com/en-us/graph/api/resources/groups-overview?view=graph-rest-1.0)."},
-			{Name: "member_ids", Type: proto.ColumnType_JSON, Hydrate: getGroupMembers, Transform: transform.FromValue(), Description: "Id of Users and groups that are members of this group."},
-			{Name: "owner_ids", Type: proto.ColumnType_JSON, Hydrate: getGroupOwners, Transform: transform.FromValue(), Description: "Id od the owners of the group. The owners are a set of non-admin users who are allowed to modify this object."},
-			{Name: "proxy_addresses", Type: proto.ColumnType_JSON, Description: "Email addresses for the group that direct to the same group mailbox. For example: [\"SMTP: bob@contoso.com\", \"smtp: bob@sales.contoso.com\"]. The any operator is required to filter expressions on multi-valued properties."},
+			{Name: "classification", Type: proto.ColumnType_STRING, Description: "Describes a classification for the group (such as low, medium or high business impact).", Transform: transform.FromMethod("GetClassification")},
+			{Name: "created_date_time", Type: proto.ColumnType_TIMESTAMP, Description: "The time at which the group was created.", Transform: transform.FromMethod("GetCreatedDateTime")},
+			{Name: "expiration_date_time", Type: proto.ColumnType_TIMESTAMP, Description: "Timestamp of when the group is set to expire.", Transform: transform.FromMethod("GetExpirationDateTime")},
+			{Name: "is_assignable_to_role", Type: proto.ColumnType_BOOL, Description: "Indicates whether this group can be assigned to an Azure Active Directory role or not.", Transform: transform.FromMethod("GetIsAssignableToRole")},
+
+			// Getting below error while requesting value for isSubscribedByMail
+			// 	{
+			// 		"error": {
+			// 				"code": "ErrorInvalidGroup",
+			// 				"message": "The requested group '[id@tenantId]' is invalid.",
+			// 				"innerError": {
+			// 						"date": "2022-07-13T11:06:23",
+			// 						"request-id": "63a83d86-a007-4c68-be75-21cea61d830e",
+			// 						"client-request-id": "d69d6667-e818-a322-c694-1fec40b438a8"
+			// 				}
+			// 		}
+			// }
+			// {Name: "is_subscribed_by_mail", Type: proto.ColumnType_BOOL, Description: "Indicates whether the signed-in user is subscribed to receive email conversations. Default value is true.", Transform: transform.FromMethod("GetIsSubscribedByMail")},
```

Comment:
Is this a breaking change, since the column looks like it was there before?

Reply:
@cbruno10, initially, I was getting some errors while trying to access this column. I've checked the SDK documentation, and in the new MSGraph SDK, it is supported only on the Get group API `GET /groups/{ID}` and returned only on `$select`. I had updated the table to add it back.